src/main/java/lotto/LottoShop.java

```diff
@@ -0,0 +1,17 @@
+package lotto;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoShop {
+    private LottoMachine lottoMachine = new LottoMachine();
+
+    public List<Lottery> buy(Money inputMoney) {
```

Comment:
클래스 이름이 LottoShop인데, 메소드 이름이 buy인게 개인적으로 조금 어색하네요. 
이 객체의 입장에서 봤을땐, 돈을 받고 List<Lottery>를 제공하는 거니까
```suggestion
    public List<Lottery> buy(Money inputMoney) {
    public LIst<Lottery> sell(Money inputMoney) {
```
아니면
```suggestion
    public List<Lottery> buy(Money inputMoney) {
    public List<Lottery> provide(Money inputMoney) {
```
가 낫지 않을까요?

Reply:
앗 그러네요... 저도 모르게 객체 입장이 아니라 제 입장만 생각했나봅니다... 감사합니다!